lib/stat_tracker.rb

```diff
@@ -134,6 +119,7 @@ def total_goals
     team_goals
   end
 
+  #helper method returns hash home/away team_id and total games played
   def total_games
```

Comment:
these will come in handy. also potential for a module refactor?. something to think about